packages/fast-components-react-base/src/toolbar/toolbar.props.ts

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import {
+    ManagedClasses,
+    ToolbarClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+import { Orientation } from "@microsoft/fast-web-utilities";
+
+export interface ToolbarManagedClasses extends ManagedClasses<ToolbarClassNameContract> {}
+export interface ToolbarUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+export interface ToolbarHandledProps extends ToolbarManagedClasses {
+    /**
+     * The toolbar children
+     */
+    children?: React.ReactNode;
+
+    /**
+     * When true, the toolbar will immediately place focus on the appropriate focusable child when the toolbar mounts
+     */
+    enableAutoFocus?: boolean;
+
+    /**
+     * The index of the child which gets focused when the toolbar initially gets focus
+     */
+    initialFocusIndex?: number;
```

Comment:
If there is a default, can we call it out here?

Reply:
"If unset the toolbar will focus on the first valid focusable child."